### PR TITLE
Problem:(fix #815)technical user may experience boredom when running CLI

### LINKED ISSUE #815: Problem: technical user may experience boredom when running CLI
CLI tools could print out ASCII version of the Blockchain cat logo designed by Carson and/or display quotes or jokes.

For inspiration, see: https://github.com/corda/corda/blob/68bfb7ff66423134ec6b9fa3f75010ddf1767099/node/src/main/kotlin/net/corda/node/internal/NodeStartup.kt#L492

### PR DESCRIPTION
**Pull request by linfeng-crypto**
Solution:
* add a  blockchain cat logo designed by Carson
* play logo and jok when run `client-cli --version` and `client-rpc` starts up and chain-abci succefully start up.
* extend `client-cli --version`'s result, add compile-time and git commit version like "2020-02-25:4c86fbc"

### CODE REVIEW

File: chain-abci/Cargo.toml [hunk 1]
```diff
@@ -14,6 +14,7 @@ sgx-test = []
 
 [dependencies]
 abci = "0.6"
+client-core = { path = "../client-core"}
```
Comment (tomtau) [inline chain-abci/Cargo.toml]: ideally `chain-abci` wouldn't depend based on client-core

File: client-core/src/logo.rs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        "My wife just found out I replaced our bed\nwith a trampoline; she hit the roof.",
+        "My boss asked me who is the stupid one, me or him?\nI said everyone knows he doesn't hire stupid people.",
+        "Don't trust atoms.\nThey make up everything.",
+        "Keep the dream alive:\nhit the snooze button.",
+        "Rest in peace, boiled water.\nYou will be mist.",
+        "When I discovered my toaster wasn't\nwaterproof, I was shocked.",
+        "Where do cryptographers go for\nentertainment? The security theatre.",
+        "How did the Java programmer get rich?\nThey inherited a factory.",
+        "Why did the developer quit his job?\nHe didn't get ar-rays."
+    ];
```
Comment (tomtau) [inline client-core/src/logo.rs]: taking other project's jokes isn't very original -- especially if they refer to the other project ("pacman logo"), they don't make sense

Comment (tomtau) [reply client-core/src/logo.rs]: better to be removed or you can come up with original jokes

Comment (linfeng-crypto) [reply client-core/src/logo.rs]: Ok

Comment (devashishdxt) [reply client-core/src/logo.rs]: @leejw51crypto Can help ;)

File: chain-core/Cargo.toml [hunk 3]
```diff
@@ -12,6 +12,8 @@ mesalock_sgx = ["sgx_tstd", "secp256k1zkp/sgx"]
 
 
 [dependencies]
+console = "0.9"
+rand = "0.7"
```
Comment (tomtau) [inline chain-core/Cargo.toml]: putting this into chain-core isn't worth either

File: client-cli/Cargo.toml [hunk 4]
```diff
@@ -26,4 +27,8 @@ cli-table = "0.2"
 zeroize = "1.1"
 base64 = "0.11"
 serde_json = "1.0"
+lazy_static = "1.4.0"
```
Comment (devashishdxt) [inline client-cli/Cargo.toml]: People have started moving away from `lazy_static` crate. You can use [`once_cell`](https://crates.io/crates/once_cell) instead.

File: chain-core/src/logo.rs [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+    let jok = joks[rng.gen_range(0, joks.len())];
+    format!("{} {}", jok.0, Emoji::new(jok.1, ""))
+}
+
+/// print out the logo and jok
+pub fn print_logo_and_jok() {
+    let logo = get_logo();
+    let jok = get_jok();
+    println!("{}\n{}", logo, jok);
+}
```
Comment (leejw51crypto) [inline chain-core/src/logo.rs]: yes, better to move "client-cli"

**Pull request merged**